pennylane/operation.py

```diff
@@ -1225,7 +1277,20 @@ def inv(self):
             self.inverse = not self._inverse
         return self
 
-    def matrix(self, wire_order=None):
+    @property
+    def matrix(self):
+        warnings.warn(
+            "The 'matrix' property is deprecated, and will be replaced with a method 'matrix()' in v0.23",
+            UserWarning,
+        )
+        op_matrix = self._matrix(*self.parameters)
```

Comment:
I'm confused...do we still HAVE `_matrix`? :) Shouldn't this call `get_matrix`?

Reply:
```suggestion
        op_matrix = self.get_matrix()
```

oops, thanks for catching this one